examples/static-file-server/src/tests.rs

```diff
@@ -0,0 +1,64 @@
+use super::*;
+use axum::http::StatusCode as SC;
```

Comment:
This is really just an opinion, but I don't like this rename. `StatusCode` is not that long of a name, it's a bit clearer to people what this is when they read the code without also reading imports and you even use both `StatusCode` and `SC` in this file. Which happens to work just because you also have `use super::*`.

This might be even more contentious opinion especially in tests, but I'd be in favor of not using `*` in imports, even here.